assets/javascripts/kitten/karl/pages/subscription-details/stories.js

```diff
@@ -0,0 +1,290 @@
+              <Text size="tiny">Le nouveau numéro</Text>
+            </li>
+            <li className="k-SubscriptionDetails__listItem">
+              <StatusIconNext status="success" color="var(--color-primary-500)" />
+              <Text size="tiny">Les coulisses de l'édition</Text>
+            </li>
+            <li className="k-SubscriptionDetails__listItem">
+              <StatusIconNext status="success" color="var(--color-primary-500)" />
+              <Text size="tiny">Un poster édition limitée</Text>
+            </li>
+            <li className="k-SubscriptionDetails__listItem">
+              <StatusIconNext status="success" color="var(--color-primary-500)" />
+              <Text size="tiny">Une surprise collector</Text>
+            </li>
+          </ul>
+        </div>
+
+      </div>
+
+      <Stroke />
+      <div className="k-SubscriptionDetails__details">
+        <div>
+          <Text size="tiny">Fréquence</Text>
+          <Text
+            tag="p"
+            weight="bold"
+            size="tiny"
+            className="k-u-margin-none"
+          >
+            Mensuel
+          </Text>
+        </div>
+        <div>
+          <Text size="tiny">Mode de livraison</Text>
+          <Text
+            tag="p"
+            weight="bold"
+            size="tiny"
+            className="k-u-margin-none"
+          >
+            Livraison
+          </Text>
+        </div>
+      </div>
+    
+      <Stroke />
+      <div className="k-SubscriptionDetails__option">
+        <Text size="tiny">Option</Text>
+        <Text
+          tag="p"
+          weight="bold"
+          size="tiny"
+          className="k-u-margin-none"
+        >
+          Végétarien
+        </Text>
+      </div>
+    </StyledWrapper>
+  )
+}
+
+export const ContributorReward = () => {
+  return (
+    <Grid>
+      <GridCol col-m="6">
+        <StyledWrapper>
+          <div className="k-SubscriptionDetails__gridWrapper">
+            <div
+              className="k-SubscriptionDetails__imageWrapper"
+              style={{ height: '159px' }}
+            >
+              <img
+                src={`/kitten-${Math.floor(Math.random() * 10)}.jpg`}
+                alt=""
+              />
+            </div>
+            <div>
+              <Text weight="regular" size="tiny">Club des lecteurs</Text>
+              <Text size="tiny" weight="light"><br />
+                par{' '}
+                <Text weight="bold" size="regular">
+                  Sick Magazine
+                </Text>
+              </Text>
+              <ul className="k-SubscriptionDetails__list">
+              <li className="k-SubscriptionDetails__listItem">
+                  <Text size="tiny">Édition Barkley.</Text>
+                </li>
+                <li className="k-SubscriptionDetails__listItem">
+                  <Text size="tiny">- Le livre Les Finisseurs (signé par les auteurs).</Text>
```

Comment:
Je suis vraiment pas fan d'ajouter `-` dans le texte. C'est aux CSS d'ajouter ça, sinon ça sera lu par les lecteurs d'écran, pour le plus grand déplaisir de nos clients aveugles

```suggestion
                  <Text size="tiny">- Le livre Les Finisseurs (signé par les auteurs).</Text>
```